### ISSUE TITLE
IllegalArgumentException with Quoted-Printable EML File

### ISSUE THREAD

**Issue by Faelean** [bbottema/simple-java-mail#419]
We have an EML File where when running the following code:

```Java
try (FileInputStream fileInputStream = new FileInputStream(emlFileName)) {	
	Email email = EmailConverter.emlToEmail(fileInputStream);	
	System.out.println(email.getHTMLText());
}
```

we get the following exception:

```
Exception in thread "main" java.lang.IllegalArgumentException: unknown content transfer encoder: QUOTED-PRINTABLE
	at org.simplejavamail.api.email.ContentTransferEncoding.lambda$byEncoder$1(ContentTransferEncoding.java:52)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at org.simplejavamail.api.email.ContentTransferEncoding.byEncoder(ContentTransferEncoding.java:52)
	at org.simplejavamail.converter.EmailConverter.buildEmailFromMimeMessage(EmailConverter.java:667)
	at org.simplejavamail.converter.EmailConverter.mimeMessageToEmailBuilder(EmailConverter.java:136)
	at org.simplejavamail.converter.EmailConverter.mimeMessageToEmailBuilder(EmailConverter.java:122)
	at org.simplejavamail.converter.EmailConverter.emlToEmailBuilder(EmailConverter.java:390)
	at org.simplejavamail.converter.EmailConverter.emlToEmailBuilder(EmailConverter.java:369)
	at org.simplejavamail.converter.EmailConverter.emlToEmail(EmailConverter.java:303)
	at org.simplejavamail.converter.EmailConverter.emlToEmail(EmailConverter.java:295)
	at ParseP7s.main(ParseP7s.java:22)
```

I've tested it with the following versions:
* 7.1.1 --> Issue is not present, Email is parsed as expected
* 7.4.0 --> Issue is present
* 7.5.0 --> Issue is present

I can not publish the example mail we're using, but if you need it I can send it directly to you.

**Comment by bbottema** [comment]
> ```java
> System.out.println(email.getHTMLText());
> ```

I'm not able to reproduce it with this email, but this issue might have actually been fixed in 7.5.1 due to this code improvement: https://github.com/bbottema/simple-java-mail/pull/416

**Comment by Faelean** [comment]
Tested it in 7.5.2 with the examples we have and the emails are parsed as expected.
Thanks for the update.
